### ISSUE TITLE
Feature request: Get new receive address

### ISSUE THREAD

**Issue by Falci** [kyokan/bob-wallet#541]
Hey!

I'd like to have a button in the `Receive` page so I can generate a new receive address, even if the previous wasn't used yet.

Rationale:
I want to transfer several domains from a popular handshake service provider but for privacy reasons I'd like to have each of them transfered to a different name. Since this service provider is slow to act, I don't want to wait until they initiate the first transfer (which would make Bob generate a new address).

<img width="1136" alt="image" src="https://user-images.githubusercontent.com/1330573/181451077-030f9ad8-9ad4-487a-a065-baed9a304b32.png">

**Comment by pinheadmz** [comment]
The only problem with this is the possibility to mess up future wallet recoveries by making too big a gap in the bip44 chain. Advanced users can already get new addresses using CLI with bob: https://hsd-dev.org/api-docs/?shell--cli#generate-receiving-address

**Comment by Falci** [comment]
My use case is risk free, since the address will be used, just by a lazy party

**Comment by pinheadmz** [comment]
yeah i have no problem with this, really. I use Wasabi wallet for BTC and it will give me as many new addresses as I want. Maybe there could simply be a 🔁  icon next to the copy button, and it will derive the next address on every click (and automatically copy it to clipboard!)

**Comment by rithvikvibhu** [comment]
How about this:
- New button that generates new address
- BUT it only works 200 times (or whatever the default lookahead value is)
- If someone _really_ needs more than 200, the API is always an option.

**Comment by brandondees** [comment]
Bump.
I'd suggest limiting it to the lookahead (or less, even) by default, or just not exposing the feature in the main UI by default, but provide a "yes i know what i'm doing and accept the risks" setting to unlock it. I like the idea of automatically copying to clipboard, especially if there's an option to generate N addresses at a time (unix-friendly newline separated values).
